StringCalculatorKata.Test/StringCalculatorTest.cs

```diff
@@ -0,0 +1,75 @@
+﻿using FluentAssertions;
+using AutoFixture;
+
+namespace StringCalculatorKata.Test
+{
+    public class StringCalculatorTest
+    {
+        [Theory]
+        [InlineData("", 0)]
+        [InlineData("1", 1)]
+        [InlineData("1,1", 2)]
+        public void AddingUpToTwoNumbers(string numbers, int actualSum)
+        {
+            AssertHelper(numbers, actualSum);
+        }
+
+        [Fact]
+        public void AddingUnknownAmountOfNumbers()
+        {
+            // Arrange
+            var fixture = new Fixture();
+            var randomNumberOfOnes = fixture.Create<int>();
+            var onesString = string.Empty;
+
+            // Act 
+            for (var i = 0; i < randomNumberOfOnes; i++)
```

Comment:
in unit tests we avoid any logic in creating test data
we simply define the input as it is, without any further logic that can lead to unexpected results.
at the end we are testing logic! we don't need to add logic in the unit of test

Reply:
Done